docs/maui/converters/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Converters - .NET MAUI Community Toolkit
+author: bijington
+description: The .NET MAUI Community Toolkit provides a collection of pre-built, reusable converters to make developers lives easier.
+ms.date: 03/13/2022
+---
+
+# Converters
+
+.NET Multi-platform App UI (.NET MAUI) data bindings usually transfer data from a source property to a target property, and in some cases from the target property to the source property. This transfer is straightforward when the source and target properties are of the same type, or when one type can be converted to the other type through an implicit conversion. When that is not the case, a type conversion must take place.
+
+For further information on Converters please refer to the [.NET MAUI documentation](/dotnet/maui/fundamentals/data-binding/converters).
+
+## .NET MAUI Community Toolkit Converters
+
+The .NET MAUI Community Toolkit provides a collection of pre-built, reusable converters to make developers lives easier. Here are the converters provided by the toolkit:
+
+| Converter | Description |
+| --------- | ----------- |
+| [`EqualConverter`](equal-converter.md) | The `EqualConverter` is a one way converter that returns a `bool` indicating whether the binding value is equal to another specified value. |
```

Comment:
OK last thing I think, not all converters that are added now are in this table

Reply:
Pesky other Shaun submitting PRs to hinder this progress. Good catch thanks